doc/ref_material/nest2_vs_3.rst

```diff
@@ -1,7 +1,7 @@
-NEST 2.X vs. NEST 3.0 conversion reference guide
-==================================================
+NEST2-to-NEST3 transition guide
```

Comment:
Should we really have the hyphens in there?

Reply:
How about ``NEST 3.0 transition guide``? I suggest keeping the top level headings short, so that they fit on one line and the table of contents is skimmable for the user.

Reply:
Good idea!

Reply:
Done.